plugins/generator-1.17.1/forge-1.17.1/procedures/world_entity_inrange.java.ftl

```diff
@@ -1,11 +1,10 @@
-((Entity) world.getEntitiesOfClass(${generator.map(field$entity, "entities", 0)}.class, new AABB(
-            ${input$x} - (${input$range} / 2d), ${input$y} - (${input$range} / 2d), ${input$z} - (${input$range} / 2d),
-            ${input$x} + (${input$range} / 2d), ${input$y} + (${input$range} / 2d), ${input$z} + (${input$range} / 2d)), e -> true)
-    .stream()
+((Entity) world.getEntitiesOfClass(${generator.map(field$entity, "entities", 0)}.class,
+	AABB.ofSize(new Vec3(${input$x}, ${input$y}, ${input$z}), ${input$range}, ${input$range}, ${input$range}), e -> true)

```

Comment:
This will not work as the old code. x y z should be in the center of the search cube, but in this case, they are in the corner

Reply:
Can't check the code rn, but I'm pretty sure this works the same as the older code

Reply:
You are right, I did not notice ofSize and thought you were using AABB constructor. It is ok :)